### PR TITLE
Template documentation update

### PR DESCRIPTION
**Pull request by zhenik**

### CODE REVIEW

**Review by Neha-Sinha2305 — COMMENTED** [review]
CHANGELOG.md file missing ;)

File: template/example/README.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+│   │   ├── outputs.tf
+│   ├── nestedB/
+│   ├── .../
+├── examples/
+│   ├── exampleA/
+│   │   ├── main.tf
+│   ├── exampleB/
+│   ├── .../
+```
+
```
Comment (pdmthorsrud) [inline template/example/README.md:37]:
```suggestion
```

Following my other comment I think this should be put it [template/README.md](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/template/README.md)

File: template/README.md [hunk 2]
```diff
@@ -78,6 +78,60 @@ vagrant ssh -c 'vault read consul/creds/admin-team'
 
 *Tokens can be used to access UI (different access level depends on role)
 
+## Minio
+Minio S3 can be used as a registry(docker registry, nexus, s3, etc...) to push, pull and store resources for further deployments.
```
Comment (fredrikhgrelland) [inline template/README.md:82]:
```suggestion
Minio S3 can be used as a general artifact repository while building and testing within the scope of the vagrantbox to push, pull and store resources for further deployments.
```

File: template/example/README.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # This is a terraform module example
+The current directory contains terraform related files that use the module in `../`.
```
Comment (pdmthorsrud) [inline template/example/README.md:2]:
```suggestion
# This is a terraform module example
The current directory contains terraform related files that use the module in `../`. See [test_example](../test_example/example/) for an example on how to do this.
```

Comment (zhenik) [reply template/example/README.md:2]: maybe just ?
```suggestion
# This is a terraform module example
The current directory contains terraform related files that use the module in `../`. See [test_example](../test_example/example/).
```

Comment (pdmthorsrud) [reply template/example/README.md:2]: Yeah, that's better! 

I like words if you can't tell :grinning:

**Comment by zhenik** [comment]
@fredrikhgrelland @pdmthorsrud , do you want me to squash all the commits into one and rebase with master?

**Review by pdmthorsrud — APPROVED** [review]

**Review by pdmthorsrud — APPROVED** [review]

**Pull request merged**